rust/private/transitions.bzl

```diff
@@ -0,0 +1,20 @@
+def _wasm_transition(settings, attr):
+    if attr.crate_type == "proc-macro":
+        return {"//command_line_option:platforms": "@local_config_platform//:host"}
+    else:
+        return settings
+
+def _wasm_bindgen_transition(settings, attr):
+    return {"//command_line_option:platforms": "@io_bazel_rules_rust//rust/platform:wasm"}
+
+wasm_transition = transition(
```

Comment:
Does it make sense to have a test for this functionality (maybe under //test/wasm_transition)?

Reply:
Perhaps although the existing `hello_world_wasm_test` does cover this because it depends on one of the wasm_bindgen dependencies. Removing the transition will cause the test to fail compilation. Do you think we need a specific test?

Reply:
Ack, marking resolved.